ctypes-foreign.opam

```diff
@@ -10,6 +10,7 @@ depexts: [
   ["libffi"] {os = "macos" & os-distribution = "macports"}
   ["libffi-devel"] {os-distribution = "centos"}
   ["libffi-devel"] {os-distribution = "ol"}
+  ["libffi" "system:pkgconf"] {os = "win32" & os-distribution = "cygwinports"}
```

Comment:
Is `"system:pkgconf"` still necessary? The package already depends on conf-pkg-config now.

Reply:
Good point; fixed